Packs/OpenLDAP/Integrations/OpenLDAP/OpenLDAP.py

```diff
@@ -20,14 +20,19 @@ class LdapClient:
         :param kwargs: Initialize params for ldap client
     """
 
+    OPENLDAP = 'OpenLDAP'
+    ACTIVE_DIRECTORY = 'Active Directory'
     GROUPS_TOKEN = 'primaryGroupToken'
     GROUPS_MEMBER = 'memberOf'
     GROUPS_PRIMARY_ID = 'primaryGroupID'
     TIMEOUT = 120  # timeout for ssl/tls socket
     DEV_BUILD_NUMBER = 'REPLACE_THIS_WITH_CI_BUILD_NUM'  # is used only in dev mode
     SUPPORTED_BUILD_NUMBER = 57352  # required server build number
+    CIPHERS_STRING = '@SECLEVEL=1:ECDHE+AESGCM:ECDHE+CHACHA20:DHE+AESGCM:DHE+CHACHA20:ECDH+AESGCM:DH+AESGCM:' \
```

Comment:
To deal with this issue in all the integrations that use HTTP communication, the CIPHERS_STRING was added to the CSP in the past (by @wissamg). 
Here we don't use HTTP, we use LDAP protocol and a raw socket - there are only two integrations that use the raw socket communication, this one, and the 'Active directory query V2' integration.
If you think it is necessary, we can add a base for them to the CSP - but I tend to leave it this way because it's only two integrations.